app/src/main/java/org/fossasia/openevent/general/OpenEventDatabase.kt

```diff
@@ -92,4 +94,6 @@ abstract class OpenEventDatabase : RoomDatabase() {
     abstract fun notificationDao(): NotificationDao
 
     abstract fun settingsDao(): SettingsDao
+
+    abstract fun texDao(): TaxDao
```

Comment:
texDao ?

Reply:
It should be taxDao*, will update it :+1:

Reply:
updated